pkg/chunk/schema_test.go

```diff
@@ -333,3 +335,55 @@ func TestSchemaRangeKey(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkEncodeLabelsJson(b *testing.B) {
```

Comment:
This was a benchmark I used to decide if I should use json or string encoding for the labels set.

`.String()`/`promql.ParseMetric` is faster than `json.Marshal`/`json.Unmarshal`.

Reply:
In case you didn't notice, `pkg/chunk` has special code to speed up the latter via `github.com/json-iterator/go`

Reply:
I'll give it a look, thanks @bboreham .

Reply:
Wow 
```
BenchmarkEncodeLabelsString-16    	    1000	     13343 ns/op	    2029 B/op	      35 allocs/op
BenchmarkEncodeLabelsJson-16    	    1000	      1074 ns/op	     586 B/op	      18 allocs/op
```

@gouthamve I see you already approved I'm going to change to use json-iterator as suggested by @bboreham

Reply:
done !